package.json

```diff
@@ -79,7 +79,7 @@
     "vuepress-theme-chartjs": "^0.2.0"
   },
   "peerDependencies": {
-    "chart.js": ">=3.2.0"
```

Comment:
Why is this necasarry, looks like this should still work fine with v3, otherwise this will be a breaking change and needs a major version bump

Reply:
Let's revert this change, and deal with possible errors with Chart.js v3 in case we encounter them.